src/shared/locales/all.js

```diff
@@ -50,151 +52,157 @@ import appLocaleZhTW from '@shared/locales/zh-TW'
 // Please keep the locale key in alphabetical order.
 /* eslint-disable quote-props */
 const resources = {
-  'ar': {
-    translation: {
-      ...eleLocaleAr,
-      ...appLocaleAr
-    }
-  },
-  'bg': {
-    translation: {
-      ...eleLocaleBg,
-      ...appLocaleBg
-    }
-  },
-  'ca': {
-    translation: {
-      ...eleLocaleCa,
-      ...appLocaleCa
-    }
-  },
-  'de': {
-    translation: {
-      ...eleLocaleDe,
-      ...appLocaleDe
-    }
-  },
-  'el': {
-    translation: {
-      ...eleLocaleEl,
-      ...appLocaleEl
-    }
-  },
-  'en-US': {
-    translation: {
-      ...eleLocaleEn,
-      ...appLocaleEnUS
-    }
-  },
-  'es': {
-    translation: {
-      ...eleLocaleEs,
-      ...appLocaleEs
-    }
-  },
-  'fa': {
-    translation: {
-      ...eleLocaleFa,
-      ...appLocaleFa
-    }
-  },
-  'fr': {
-    translation: {
-      ...eleLocaleFr,
-      ...appLocaleFr
-    }
-  },
-  'hu': {
-    translation: {
-      ...eleLocaleHu,
-      ...appLocaleHu
-    }
-  },
-  'id': {
-    translation: {
-      ...eleLocaleId,
-      ...appLocaleId
-    }
-  },
-  'it': {
-    translation: {
-      ...elelocaleIt,
-      ...applocaleIt
-    }
-  },
-  'ja': {
-    translation: {
-      ...eleLocaleJa,
-      ...appLocaleJa
-    }
-  },
-  'ko': {
-    translation: {
-      ...eleLocaleKo,
-      ...appLocaleKo
-    }
-  },
-  'nb': {
-    translation: {
-      ...eleLocaleNb,
-      ...appLocaleNb
-    }
-  },
-  'pl': {
-    translation: {
-      ...eleLocalePl,
-      ...appLocalePl
-    }
-  },
-  'pt-BR': {
-    translation: {
-      ...eleLocalePtBR,
-      ...appLocalePtBR
-    }
-  },
-  'ro': {
-    translation: {
-      ...eleLocaleRo,
-      ...appLocaleRo
-    }
-  },
-  'ru': {
-    translation: {
-      ...eleLocaleRu,
-      ...appLocaleRu
-    }
-  },
-  'tr': {
-    translation: {
-      ...eleLocaleTr,
-      ...appLocaleTr
-    }
-  },
-  'uk': {
-    translation: {
-      ...eleLocaleUk,
-      ...appLocaleUk
-    }
-  },
-  'vi': {
-    translation: {
-      ...eleLocaleVi,
-      ...appLocaleVi
-    }
-  },
-  'zh-CN': {
-    translation: {
-      ...eleLocaleZhCN,
-      ...appLocaleZhCN
-    }
-  },
-  'zh-TW': {
-    translation: {
-      ...eleLocaleZhTW,
-      ...appLocaleZhTW
-    }
-  }
-}
-/* eslint-enable quote-props */
+        'ar': {
```

Comment:
Please adjust the code indentation to 2 spaces.

Reply:
whoops. sorry. Is now done.